packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -224,12 +224,6 @@ export abstract class FluidDataStoreContext
 	public get containerRuntime(): IContainerRuntimeBase {
 		return this._containerRuntime;
 	}
-
-	// back-compat, to be removed in 2.0
-	public ensureNoDataModelChanges<T>(callback: () => T): T {
```

Comment:
Need to be deprecated first before removing

Reply:
I think this one can also be an exception since it is dependent on the deprecated version in runtime-definitinos@jason-ha Can you confirm?